samples/snippets/src/main/java/com/example/datastore/RegionalEndpoint.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2023 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.datastore;
+
+// Imports the Google Cloud client library
+
+import com.google.cloud.datastore.Datastore;
+import com.google.cloud.datastore.DatastoreOptions;
+
+public class RegionalEndpoint {
+
+  public Datastore createClient() throws Exception {
+    // Instantiates a client
+    // [START datastore_regional_endpoint]
```

Comment:
nit: current best practice is to include `import` statements and the sample class inside the region tags.

See:
https://googlecloudplatform.github.io/samples-style-guide/#region-tags

Reply:
Done.